### DISCUSSION TITLE
the file macos-13-arm64-Readme.md exist but how do I use that Image

### DISCUSSION THREAD

**Discussion by JoacimWall** [actions/virtual-environments#8749] [General questions] [answered]
Hi 
https://github.com/actions/runner-images/tree/macos-13/20231025.2/images/macos

I can't find the name for the Image in any docs.  
- job: Job
    pool:
      vmImage: 'macOS-13-arm64'

What should the vmImage name be to use the Arm64 version of Mac OS?

//Thanks Joacim

**Comment by shamil-mubarakshin** [comment, marked as answer]
Hi @JoacimWall,
macOS13 arm64 agent's availability date for Azure DevOps is unknown. You can use [supported](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/osx-agent?view=azure-devops#check-prerequisites) self-hosted agents.
For GitHub Actions arm64 runners are [available](https://github.com/actions/runner-images/issues/8439) by `runs-on: macos-13-xlarge`.

**Reply by peterwaller-arm** [reply]
It's unclear from the documentation what architecture you get or how you specify the architecture.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Is it not currently possible to specify the architecture independently of the size? This is a bit confusing if you need to select a specific architecture.
